TimeService/work/timeservice/implem/default/C/src/timeservice-api.c

```diff
@@ -0,0 +1,110 @@
+#include "timeservice-api.h"
+
+#define BYTE_CAPACITY 256
+#define CUC_TFIELD_SIZE 7
+
+asn1SccCfsTimestamp nanoseconds_to_cfs_timestamp(uint64_t nanoseconds)
+{
+    asn1SccCfsTimestamp timestamp;
+    timestamp.seconds = nanoseconds / NS_PER_SECOND;
+    timestamp.subseconds = nanoseconds % NS_PER_SECOND;
```

Comment:
This should be a binary, not decimal fraction.

Reply:
Changed from decimal to binary divisor (2^32) as per documentation